consensus/quorum/quorum.go

```diff
@@ -217,7 +217,35 @@ func (s *cIdentities) NthNext(pubKey *bls.PublicKeyWrapper, next int) (bool, *bl
 	return found, &s.publicKeys[idx]
 }
 
-// NthNextHmy return the Nth next pubkey of Harmony nodes, next can be negative number
+// NthNextValidatorHmy return the Nth next pubkey nodes, but from another validator.
+func (s *cIdentities) NthNextValidatorHmy(instance shardingconfig.Instance, slotList shard.SlotList, pubKey *bls.PublicKeyWrapper, next int) (bool, *bls.PublicKeyWrapper) {
+	found := false
+
```

Comment:
it would be good if you validate next here:
```golang
if next <= 0 {
      return false, nil, fmt.Errorf("invalid ...")
}
```

Reply:
added check